pyqtgraph/Qt/__init__.pyi

```diff
@@ -5,32 +5,40 @@ This stub file is to aid in the PyCharm auto-completion of the Qt imports.
 from typing import Union
 
 try:
-    from PyQt5 import QtCore, QtGui, QtWidgets
+    from PyQt5 import QtCore, QtGui, QtSvg, QtTest, QtWidgets
 
     QtCore = QtCore
     QtGui = QtGui
     QtWidgets = QtWidgets
+    QtTest = QtTest
+    QtSvg = QtSvg
 except ImportError:
     try:
-        from PyQt6 import QtCore, QtGui, QtWidgets
+        from PyQt6 import QtCore, QtGui, QtSvg, QtTest, QtWidgets
 
         QtCore = QtCore
         QtGui = QtGui
         QtWidgets = QtWidgets
+        QtTest = QtTest
+        QtSvg = QtSvg
     except ImportError:
         try:
-            from PySide2 import QtCore, QtGui, QtWidgets
+            from PySide2 import QtCore, QtGui, QtSvg, QtTest, QtWidgets
 
             QtCore = QtCore
             QtGui = QtGui
             QtWidgets = QtWidgets
+            QtTest = QtTest
+            QtSvg = QtSvg
         except ImportError:
             try:
-                from PySide6 import QtCore, QtGui, QtWidgets
+                from PySide6 import QtCore, QtGui, QtSvg, QtTest, QtWidgets
```

Comment:
Curious what the motivation here is for these extra imports? I'm not seeing them referenced elsewhere in the PR.

Reply:
I noticed while looking at a test that the `pyi` file was incomplete; we already import both of these in the actual `Qt/__init__.py` file. Though, now that I'm looking at it closer, they're both imported in blocks of
```python
try:
    from PySide6 import QtTest
except ImportError as err:
    QtTest = FailedImport(err)
```
I don't know why `QtTest` wouldn't be available for import, but I'm guessing that maybe there are stripped-down distributions that don't include it? I don't know what that should mean for the `pyi` file, though. Probably any development environment is going to include the full qt library, which is what this file is intended for. However, the type-checking is going to run everywhere, regardless, and might throw an `ImportError`. Maybe I should just skip this and accept the lack of type hints... Thoughts?

Reply:
Huh, `QtTest` is shown as being present in `PySide6-essentials`   https://pypi.org/project/PySide6-Essentials/

That's weird it's not included, would it be under a different module or something?

Reply:
the import worked fine on my machine just now?

```
In [1]: from PySide6 import QtTest

In [2]: import PySide6

In [3]: PySide6.__version__
Out[3]: '6.6.2'

In [4]:
```

Reply:
okay then, I say we tentatively go forward with importing like this it in the `pyi` and expecting that nothing will blow up. I'll put in an issue to clean up `Qt.py` to keep it isolated.